Objects/namespaceobject.c

```diff
@@ -2,6 +2,9 @@
 
 #include "Python.h"
 #include "pycore_namespace.h"     // _PyNamespace_Type
+#include "pycore_global_strings.h" // _Py_ID
+#include "pycore_global_objects.h" // _Py_ID
+#include "pycore_runtime.h"       // _Py_ID
```

Comment:
"pycore_runtime.h" includes "pycore_global_objects.h", which in turn includes "pycore_global_strings.h":

```suggestion
#include "pycore_runtime.h"       // _Py_ID
```

Reply:
Oh, Victor changed this yesterday in 0dd3fc2a640b273979f94299b545e1e40ac0633c (gh-108228)

Reply:
Oh, well, "pycore_runtime.h" includes "pycore_interp.h" which includes "pycore_global_object.h", so even after Victor's clean-up, you should be fine with only one include:

```suggestion
#include "pycore_runtime.h"       // _Py_ID
```

Reply:
Is it generally preferred to minimize includes like this and rely on transitive includes? I generally thought it was better to directly include anything you need, and not consider the set of includes in another header as part of its public contract.

Reply:
I'm not sure if there is a preferred way of doing includes.